android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -45,8 +45,8 @@
 		"startBias": ["Coast"],
 		"preferredVictoryType":"Diplomatic",
 
-		"startIntroPart1": "",
-		"startIntroPart2": "",
+		"startIntroPart1": "Great King Alexander, leader of the mighty Greek nation, you are truly blessed by the gods. By your astonishing conquests, you showed once again that union leads to greatness. In the past, quarrelsome city-states such as Athens, Sparta, Thebes, found a way to join their forces, allowing Greece to defeat the mighty Persia in the 5th century BC, both on land and sea. Democracy, philosophy, tragedy, art and architecture then spread to the world, given therefore birth to the Western Civilisation.",
```

Comment:
The victory open Persia seems unrelated
Maybe "From Europe to Africa to India, you built an empire unmatched by any before it."

Reply:
> The victory open Persia seems unrelated
I want to say that Alexander followed the example of Athens, Sparta, Thebes who united their forces to defend against Persia invasion, and defeat them (480 BC).
Alexander did the same in 334 BC: he founded a coalition (Macedonia and Greece cities) to invade Persia, and strike it down.

I rewrite it, keeping your suggestion.